src/core/server/integration_tests/config/config_deprecation.test.ts

```diff
@@ -26,14 +26,15 @@ describe('configuration deprecations', () => {
   });
 
   if (getFips() === 0) {
-    it('should not log deprecation warnings for default configuration', async () => {
+    it('should log one warning for default configuration, the http/tls deprecation warning', async () => {
```

Comment:
Is it ok to have deprecation warnings on default config? Seems ok since the issue is to log when http2 or tls are not enabled but tls can't be enabled by default

Reply:
Generally we wouldn't want to log a deprecation right "out of the box" after installing Kibana. But given that TLS is such an important security configuration I think this is fine